### ISSUE TITLE
Patching & new release for image 3.3.4-nginx-202304160904

### ISSUE THREAD

**Issue by matthiasbaldi** [coreruleset/modsecurity-crs-docker#153]
When I test your image with trivy I get currently a bunch (and increasing) number of CVE issues:

```text
owasp/modsecurity-crs:3.3.4-nginx-202304160904 (debian 11.6)
Total: 430 (UNKNOWN: 5, LOW: 199, MEDIUM: 113, HIGH: 106, CRITICAL: 7)
```
```text
trivy --version
Version: 0.43.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-07-10 06:10:24.469485904 +0000 UTC
  NextUpdate: 2023-07-10 12:10:24.469485404 +0000 UTC
  DownloadedAt: 2023-07-10 08:53:49.016447862 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-07-10 00:56:20.726840625 +0000 UTC
  NextUpdate: 2023-07-13 00:56:20.726840125 +0000 UTC
  DownloadedAt: 2023-07-10 08:54:13.951526052 +0000 UTC
```

Would it be possible to build and release a new image tag, the current one is 3 months old.
May then there are already some fixed issues from the upstream image.

Thank you.

**Comment by fzipi** [comment]
Sure. Let's release.

**Comment by theseion** [comment]
The release is in progress. New nginx images should appear shortly. Please report back on what you find.

**Comment by matthiasbaldi** [comment]
Thank you.
I run it again trough trivy (same version may, updated databases)

`trivy image owasp/modsecurity-crs:3.3.4-nginx-202307110507`

```text
owasp/modsecurity-crs:3.3.4-nginx-202307110507 (debian 11.7)
Total: 354 (UNKNOWN: 3, LOW: 196, MEDIUM: 76, HIGH: 74, CRITICAL: 5)
```

I think some of the critical stuff is still coming from the upstream debian image.

**Comment by theseion** [comment]
Thanks. It's common to see critical warnings when using Debian. There's not much we can do really. I just scanned `ngins:1.24.0`, which we are currently using and that's exactly what trivy reports.

What we maybe should do is to create scheduled releases, e.g. once per week. @fzipi?

**Comment by matthiasbaldi** [comment]
ya that is possible.
It is a bit sad, that Nginx do not build images based on the xy-slim Debian images.
The slim is not so small like an Alpine (sometimes with SSL issues) but has much less CVE issues.

There are slim Images documented here https://hub.docker.com/_/nginx (title: "nginx:<version>-slim") but I couldn't find any current tags.

**Comment by fzipi** [comment]
Well, I think trivy might not be doing a good job, lately (see https://github.com/nginxinc/docker-nginx/issues/774 for example).

Using docker scout I get way different results:
```
❯ docker scout cves owasp/modsecurity-crs:3.3.4-nginx-202307110507
INFO New version 0.19.0 available (installed version is 0.16.1)
    ✓ Provenance obtained from attestation
    ✓ SBOM of image already cached, 246 packages indexed
    ✗ Detected 24 vulnerable packages with a total of 49 vulnerabilities

   0C     0H     0M     8L  tiff 4.2.0-1+deb11u4
pkg:deb/debian/tiff@4.2.0-1+deb11u4?os_distro=bullseye&os_name=debian&os_version=11
```

**Comment by matthiasbaldi** [comment]
Ya I saw such results before. But thought it is because of different sources.
But when I search through the trivy issues list I am able to find similar issues, in this case the severity is not defined according the same rules: https://github.com/aquasecurity/trivy/issues/3705.

Releasing for every nginx image a new crs image is for sure not a bad thing :+1:

**Comment by ne20002** [comment]
The new 3.3.5 ist out for two weeks now but there is no updated image on hub.docker.com.
:(

**Comment by theseion** [comment]
Thanks for the hint @ne20002. We did prepare the changes but forgot to make the release. I've created the PR.

**Comment by fzipi** [comment]
We released 3.3.5 already. Closing this one.

**Comment by matthiasbaldi** [comment]
Thank you @fzipi 
This would be the image [3.3.5-nginx-202308071108](https://hub.docker.com/layers/owasp/modsecurity-crs/3.3.5-nginx-202308071108/images/sha256-fe0936d11dd78eb2000d9eeed10cfc5f57c7b2ce483897619112015e45282f1c?context=explore)
